src/style/app.module.css

```diff
@@ -513,7 +532,11 @@
 }
 
 .searchButton {
+  --bs-btn-active-color: var(--search-button-bg);
+  --bs-btn-active-bg: var(--search-button-bg);
+  --bs-btn-active-border-color: var(--search-button-bg);
   margin-bottom: 10px;
+  color: var(--brown-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`